src/js/index.js

```diff
@@ -1,25 +1,47 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import '../style.css';
-import Score from './score.js';
 
-class ScoreArray {
-  static returnArray() {
-    const scoreArray = JSON.parse(localStorage.getItem('scoreArray') || '[]');
-    return scoreArray;
-  }
-
-  static saveScore(score) {
-    const scoreArray = ScoreArray.returnArray();
-    scoreArray.push(score);
-    localStorage.setItem('scoreArray', JSON.stringify(scoreArray));
-  }
+async function scoreArray() {
```

Comment:
- [ ] As pointed in the previous review, you need to use `arrow functions` instead of the `function keyword` for all your functions as it is a requirement in this project. Your code is working correctly as expected, don't mind, this is just an opportunity for you to learn how to use proper ES6 syntax. It's probably an omission but you should have been requested this as mandatory in the previous milestone as you can see here in the project Learning objectives.

![Screenshot from 2021-10-06 20-08-28](https://user-images.githubusercontent.com/40503704/136268802-2ce125ee-b238-4c95-94fc-e6d9afe11e85.png)